### PR TITLE
Replace Switfmailer with Symfony Mailer

### PR DESCRIPTION
**Pull request by ernestWarwas**

### CODE REVIEW

**Comment by vasilvestre** [comment]
I'm not sure but your jwt keys should be renewed because you use another algorithm

**Comment by ernestWarwas** [comment]
@vasilvestre Looks like it's not necessary 🎸

**Comment by vasilvestre** [comment]
> @vasilvestre Looks like it's not necessary guitar

Nice ! :D

**Review by Rafikooo — APPROVED** [review]

**Review by NoResponseMate — APPROVED** [review]

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -180,8 +180,12 @@ jobs:
...
 
             -
-                name: Run non-JS Behat (without managing catalog promotion scenarios)
-                run: vendor/bin/behat --colors --strict --no-interaction -vvv -f progress --tags="~@javascript&&~@todo&&~@cli&&@~managing_catalog_promotions"
+                name: Run managing exchange rates scenarios
+                run: vendor/bin/behat --colors --strict --no-interaction -vvv -f progress --tags="~@javascript&&~@todo&&~@cli&&@managing_exchange_rates"
+
+            -
+                name: Run non-JS Behat (without managing catalog promotion scenarios and exchange rates)
+                run: vendor/bin/behat --colors --strict --no-interaction -vvv -f progress --tags="~@javascript&&~@todo&&~@cli&&@~managing_catalog_promotions&&@~managing_exchange_rates"
```
Comment (GSadee) [inline .github/workflows/build.yml:188]:
```suggestion
                run: vendor/bin/behat --colors --strict --no-interaction -vvv -f progress --tags="~@javascript&&~@todo&&~@cli&&~@managing_catalog_promotions&&~@managing_exchange_rates"
```

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thank you, Ernest! :tada:

File: .github/workflows/build.yml (hunk 1 above)
Comment (lecajer) [reply .github/workflows/build.yml:188]: How is that releated to the replacement of SwiftMailer by SymfonyMailer ?

**Pull request merged**